features/support/vcr.rb

```diff
@@ -1,11 +1,17 @@
 require 'vcr'
 
+dotenv = File.join(Rails.root, '.env')
```

Comment:
This was the bit I was wondering about

Reply:
Ah, that was me just trying to be clever about filtering out any environment variables (in this case, the AWS stuff). In the past I've just filtered out anything in the `ENV` hash, but this can sometimes have unintended consequences when environments differ (for example between our local machines and CI)

Reply:
So, what this does is if there's a `.env` file present, it loads only those variables, rather than getting the whole of the local machines ENV state